build_helpers/cssTransformLoader.js

```diff
@@ -14,11 +14,6 @@ function slashTransform(content) {
 }
 
 module.exports = function(content) {
-  if (this && this.cacheable) {
```

Comment:
Seems that loaders are cacheable by default now [https://webpack.js.org/guides/migrating/#cacheable](url)